charts/prometheus-artifactory-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.14.0"
```

Let's bump the app version to the [new release](https://github.com/peimanja/artifactory_exporter/releases/tag/v1.15.0) which has your changes
```suggestion
appVersion: "1.15.0"
```